tox.ini

```diff
@@ -4,14 +4,13 @@ envlist = py27-django111,py35-django{111,22}
 [testenv]
 envdir=
     # Use the same environment for all commands running under a specific python version
-    py27: {toxworkdir}/py27
     py35: {toxworkdir}/py35
+    py38: {toxworkdir}/py38
 
 passenv =
     TEST_INTEGRATION
     TEST_DATA
 deps =
-    django111: -r requirements/django.txt
     django22: -r requirements/django22.txt
```

Comment:
Remove this line and run `make upgrade `in order to have django in the tests.txt file